src/firebase/firestore-service.ts

```
@@ -0,0 +1,114 @@
+          share: action === ACTIONS.SHARE ? 1 : 0,
+          download: action === ACTIONS.DOWNLOAD ? 1 : 0
+        });
+
+        return {
+          status: 201, 
+          success: true,
+          message: 'Story and browser record created',
+          action,
+          storyId:storyId,
+          diff: 1
+        };;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)